pkg/utils/cr2jinja2/v2alpha3/cr2jinja2.go

```diff
@@ -1,4 +1,4 @@
-package cr2jinja2
+package v2alpha3
```

Comment:
@gaohoward Is this due to the fact that we are not yet executing cr2jinja2 as a separate executable?

Reply:
yes we directly call the cr2jinja2 api to generate the content. Will think about how to fix that.

Reply:
So the reason to split the code in different version is that we have now 2 versions of crd that uses this tool, in future more versions will be added. New versions may have new address-settings properties so the tool need to deal with it.
However a lot code is repeated in this way. I'll try refactor that to use one version to deal with different versions of address-settings.

Reply:
@gaohoward Not sure if we need to tackle this right away, but in the near term after the next release this would be good.